include/deal.II/base/function_parser.h

```diff
@@ -26,10 +26,7 @@
 #include <vector>
 #include <map>
 
-namespace mu
-{
-  class Parser;
-}
+#include <muParser.h>
```

Comment:
It works because `std::vector<T>` just stores a pointer to an array of `T`s. The compiler doesn't need to know the size of `T` to determine the size of `std::vector<T>` objects.

It is of course unfortunate that MSVC doesn't recognize this :-( It requires us to pull in another header file from external :-((